pkg/psmdb/mongos.go

```diff
@@ -450,9 +450,14 @@ func MongosServiceSpec(cr *api.PerconaServerMongoDB, podName string) corev1.Serv
 	switch cr.Spec.Sharding.Mongos.Expose.ExposeType {
 	case corev1.ServiceTypeNodePort:
 		spec.Type = corev1.ServiceTypeNodePort
-		spec.ExternalTrafficPolicy = "Local"
-		if cr.CompareVersion("1.19.0") >= 0 {
-			spec.ExternalTrafficPolicy = "Cluster"
+		if len(cr.Spec.Sharding.Mongos.Expose.ExternalTrafficPolicy) != 0 {
```

Comment:
@nmarukovich do we have this option in our cr.yaml? please add it

Reply:
I forgot to commit cr 
I will fix it